src/Commons/Travely.Common.ServiceDiscovery/ServiceDiscoveryExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Consul;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+
+namespace Travely.Common.ServiceDiscovery
+{
+    public static class ServiceDiscoveryExtensions
+    {
+        public static void RegisterConsulServices(this IServiceCollection services, IConfiguration configuration)
```

Comment:
I would change to Microsoft's style:
public static IServiceCollection AddConsulServices(this IServiceCollection services, IConfiguration configuration)

Reply:
Done